schemas/ohi.yml

```diff
@@ -59,6 +67,31 @@
         - bullseye
         - bookworm
 
+- src: "{app_name}_{version}-1_{arch}_fips.deb"
```

Comment:
I wonder if the `fips` should be part of the `app_name` and not a suffix.

i.e. app_name=`newrelic-infra-fips`

https://www.debian.org/doc/manuals/debian-reference/ch02.en.html#_debian_package_file_names

Same for `rpm`. Have you already discussed this?

Reply:
No, this has not been discussed. But I agree with you. `fips` in the name before `arch` makes sense.

Reply:
I'll tell Dan also to do the change.

Reply:
Notified for the infra-agent PR in here https://github.com/newrelic/infrastructure-agent/pull/1942#pullrequestreview-2451394416

Reply:
Done, thanks for pointing out.